src/transformers/generation_utils.py

```diff
@@ -191,6 +191,7 @@ class BeamSearchDecoderOnlyOutput(ModelOutput):
             Processed beam scores for each vocabulary token at each generation step. Beam scores consisting of log
             softmax scores for each vocabulary token and sum of log softmax of previously generated tokens in this beam
             . :obj:`(max_length,)`-shaped tuple of :obj:`torch.FloatTensor` with each tensor of shape :obj:`(batch_size
+
```

Comment:
Ugh, another failure of the new doc styler. Can we use no space for the multiplications so that it doesn't believe this a listt?

Reply:
yes will change it :-)